### PR TITLE
Adding support for Publish Test results to .Net core task

### PR DESCRIPTION
**Pull request by nigurr**

### CODE REVIEW

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 1]
```diff
@@ -91,6 +89,58 @@ export class dotNetExe {
         }
     }
 
+    private async executeTestCommand(): Promise<void> {
+        const dotnetPath = tl.which('dotnet', true);
+        const enablePublishTestResults: string = tl.getInput('publishTestResults', false) || 'false';
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: consider using getBoolInput function in task-lib (tl)

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 2]
```diff
@@ -91,6 +89,58 @@ export class dotNetExe {
...
+        this.extractOutputArgument();
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: You don't need this since you are not using this

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 3]
```diff
@@ -91,6 +89,58 @@ export class dotNetExe {
         }
     }
 
+    private async executeTestCommand(): Promise<void> {
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: consider adding UTs for this change

Comment (nigurr) [reply Tasks/DotNetCoreCLI/dotnetcore.ts]: Added

File: Tasks/DotNetCoreCLI/dotnetcore.ts [hunk 4]
```diff
@@ -91,6 +89,57 @@ export class dotNetExe {
...
+        const dotnetPath = tl.which('dotnet', true);
+        const enablePublishTestResults: boolean = tl.getBoolInput('publishTestResults', false) || false;
+        const resultsDirectory = tl.getVariable('Agent.TempDirectory');
+
+        if (enablePublishTestResults && enablePublishTestResults === true) {
```
Comment (sachinma) [inline Tasks/DotNetCoreCLI/dotnetcore.ts]: You can just use enablePublishTestResults now

**Pull request merged**